consistency-tests/__tests__/clickables.test.tsx

```diff
@@ -54,6 +54,8 @@ describe.each`
     });
 
     afterEach(() => {
+        // @ts-expect-error [FEI-5019] - TS2339 - Property 'mockClear' does not exist on type '(url: string | URL) => void'.
```

Comment:
I see we have other similar `@ts-expect-error`s in the repo so this is probably fine! I wonder if there's an alternative way to address these TS issues 🤔 

Interesting to see we have `@ts-expect-error`s like this in `frontend` too!